### PR TITLE
Nodes Cards

### LINKED ISSUE #400: Nodes Cards Implementation
Cards for different node types in lacchain 

Node Types:

- Validator 
- Boot
- Writer
- Oberserver

image

### PR DESCRIPTION
**Pull request by tetogomez**
### What does this PR do?

- Resolve #400 

### Steps to test

1. first step
2. next step

#### CheckList

- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: webapp/src/components/InformationCard/Social.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import Box from '@material-ui/core/Box'
+
+import ProducerSocialLinks from '../ProducerSocialLinks'
+
+const Social = ({ social, type, t }) => {
+  if (type === 'node') return <></>
+
+  return (
+    <Box className="social">
+      <Typography variant="overline">Social</Typography>
```
Comment (xavier506) [inline webapp/src/components/InformationCard/Social.js:13]: should we use i18n?

File: webapp/src/components/InformationCard/index.js [hunk 2]
```diff
@@ -9,184 +9,107 @@ import Collapse from '@material-ui/core/Collapse'
...
+                nodes={producerOrg.nodes}
+                producer={producer}
+                t={t}
+                onNodeClick={onNodeClick}
+                type={type}
+              />
             </Box>
             <Box className={classes.twoBoxes}>
               <Box className="healthStatus">
                 <Typography variant="overline">Health</Typography>
```
Comment (xavier506) [inline webapp/src/components/InformationCard/index.js:103]: i18n?

File: webapp/src/components/InformationCard/Social.js (hunk 1 above)
Comment (tetogomez) [reply webapp/src/components/InformationCard/Social.js:13]: yes, on it!

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by adriexnet — APPROVED** [review]

**Pull request merged**